package.json

```diff
@@ -48,23 +51,22 @@
     "jest": "^27.4.2",
     "jest-axe": "^6.0.0",
     "npm-run-all": "4.1.5",
+    "react-redux": "^7.2.2",
```

Comment:
Just noticed this because it moved.  Do we really need redux?  I don't see anywhere it is used.

Reply:
I think react-redux is a peer dependency of the SDK - though I think that is enforced on hac-core, not hac-infra